hoodie-spark/src/test/java/HoodieJavaApp.java

```diff
@@ -44,52 +44,53 @@
  */
 public class HoodieJavaApp {
 
-  @Parameter(names = {"--table-path", "-p"}, description = "path for Hoodie sample table")
-  private String tablePath = "file:///tmp/hoodie/sample-table";
+  public static class Config extends AbstractCommandConfig {
```

Comment:
Lets also revert this change as AbstractCommandConfig will be in hoodie-utilities